go-selinux/selinux_linux.go

```diff
@@ -445,6 +460,186 @@ func ComputeCreateContext(source string, target string, class string) (string, e
 	return readWriteCon(filepath.Join(getSelinuxMountPoint(), "create"), fmt.Sprintf("%s %s %d", source, target, classidx))
 }
 
+/*
+catsToBitset stores categories in a bitset.
+*/
+func catsToBitset(cats string) (*bitset.BitSet, error) {
+	bitset := &bitset.BitSet{}
+
+	catlist := strings.Split(cats, ",")
+	for _, r := range catlist {
+		ranges := strings.Split(r, ".")
+		if len(ranges) > 1 {
+			catstart, err := strconv.Atoi(string([]rune(ranges[0])[1:]))
```

Comment:
I think you should use  strconv.ParseUint(, 10, 64) In order to catch bogus numbers.